.gitignore

```diff
@@ -13,7 +13,6 @@ build.properties
 
 # Composer
 vendor
-composer.lock
```

Comment:
The lock file is what guarantees everyone is working from the same codebase. Without it, you would need to run `composer update` which could result in one install being different from another. Generally,  only one member would ever run `composer update`. Everyone else would only run `composer install`.